### ISSUE TITLE
Make required permissions visible

### ISSUE THREAD

**Issue by maxandersen** [giscus/giscus#314]
Currently when logging in to GitHub it just asks “act on your behalf” without stating it’s just to get username and allow to post a comment. 

This have made people In quarkus community raise concerns as you don’t actually know what you agree to. 

I know GitHub is notoriously bad at showing permission clearly but I’m wondering if possible to add showing the needed permission into the login flow - maybe as a pop up before going to GitHub ?

**Comment by laymonage** [comment]
I don't think a pop up before going to GitHub will make much difference for the skeptics. I mean, a malicious app can also show a pop up that says "we're totally safe!!", as it's done by the app itself.

GitHub actually shows the detailed permissions after you authorize the app.

![image](https://user-images.githubusercontent.com/6379424/146296267-9fbb40b7-e631-4856-9166-d4b0ca196e92.png)

I think [this docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/authorizing-github-apps) should answer their questions. I doubt there's much else I can do to assure them.

**Comment by maxandersen** [comment]
I think what you are showing is for the giscus.app that the blog/site owner agrees to; I mean for the "login" needed as user says "Act on your behalf":

<img width="522" alt="2021-12-16_09-27-59" src="https://user-images.githubusercontent.com/54129/146335417-34dbc7c6-54b5-4097-9a87-321012bd7cf0.png">

that raises some eyebrows as it reads as if giscus.app can now do *anything* not just post comments.

**Comment by laymonage** [comment]
Yeah... I'm not sure if adding a pop-up would help much, though. The option suggested in #315 might be more helpful.

I guess I need to add a link to this docs somewhere: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/authorizing-github-apps

**Comment by maxandersen** [comment]
Or "Learn more about giscus" which has content you can add to afaik ? Here might list it just do comments.

But yeah I think the real power of giscus approach shines with #315.
